.github/workflows/backwards_compatibility_marqo_execution.yml

```diff
@@ -0,0 +1,167 @@
+# Execution workflow
+name: Marqo Compatibility Tests Execution
+
+on:
+  workflow_call:
+    inputs:
+        from_version:
+            description: 'Source Marqo version'
+            required: true
+            type: string
+        to_version:
+            description: 'Target Marqo version'
+            required: true
+            type: string
+        to_version_digest:
+            description: 'Target Marqo version digest'
+            required: true
+            type: string
+        from_image:
+            description: 'Source Marqo image (optional)'
+            required: false
+            type: string
+        to_image:
```

Comment:
What happens to this if digest is also provided? I'm not sure if the distinction matters for input parameters because input can be image:tag or image@digest and all commands should work with both

Reply:
Currently to_image is just specified as a input parameter, it's passed as an input to compatibility_test_runner.py but compatibility_test_runner.py doesn't process it. I've added a TODO to start using it in the future, I've also created a ticket for tracking: https://s2search.atlassian.net/browse/MQ-274, as this is non-blocking, I will address it in the next PR

Reply:
Sounds good

Reply:
Sorry this still doesn't make sense to me. Why can't we have one input arg that can be a tag or digest?

Reply:
Discussed offline